src/enzyme_ad/jax/enzyme_call.cc

```diff
@@ -868,7 +868,7 @@ class CpuKernel {
     // force deletion of mod first explicitly
     mod = nullptr;
     return std::make_pair(res, tmpBuf);
-  }
+  }  
```

Comment:
Just a small formatting request
```suggestion
  }
```